doc/tutorials/core/univ_intrin/univ_intrin.markdown

```diff
@@ -0,0 +1,337 @@
+Vectorizing your code using Universal Intrinsics {#tutorial_univ_intrin}
+==================================================================
+
+@tableofcontents
+
+@prev_tutorial{tutorial_how_to_use_OpenCV_parallel_for_new}
+
+|    |    |
+| -: | :- |
+| Compatibility | OpenCV >= 3.0 |
+
+Goal
+----
+
+The goal of this tutorial is to provide a guide to using the Universal Intrinsics feature to vectorize your C++ code for a faster runtime.
+We'll briefly look into _SIMD intrinsics_ and how to work with wide _registers_, followed by a tutorial on the basic operations using wide registers.
+The tutorial will only demonstrate basic operations. To know more about Universal Intrinsics, visit the [documentation](https://docs.opencv.org/4.5.3/df/d91/group__core__hal__intrin.html).
```

Comment:
> https://docs.opencv.org/...

ditto

Reply:
How do I refer to the documentation?

Reply:
Use `@ref <id>`, where `<id>`:

- is a tutorial identifier (see the first line of tutorial page)
- or identifier of code group, for this case it is [here](https://github.com/opencv/opencv/blob/4.5.3/modules/core/include/opencv2/core.hpp#L93)